pysgrid/tests/test_sgrid.py

```diff
@@ -398,22 +393,14 @@ def test_node_dimesnions(sgrid_wrf_sgrid_2d):
     assert node_dims == expected
 
 
-# TestSGridSaveNodeCoordinates
-"""
-Test that SGrid.save_as_netcdf is saving
-content correctly.
+# Test that SGrid.save_as_netcdf is saving
+# content correctly.
 
-There maybe a better way to do this using
-mocks, but this will do for now.
-
-"""
-
-
-def round_trip_deltares(deltares_sgrid):
+def test_round_trip_deltares(deltares_sgrid):
```

Comment:
I am surprised these passed. I added them just a place holder until I finished the round-trip (navigating the attributes from the saved file and the original).